docs/content/latest/explore/indexes-constraints/indexes-1.md

```diff
@@ -1,16 +1,15 @@
 ---
 title: Indexes
 linkTitle: Indexes
-description: Using indexes
-headcontent: Indexes
+description: Using Indexes
 image: /images/section_icons/secure/create-roles.png
 menu:
   latest:
     identifier: indexes-1
-    parent: explore-ysql-language-features
-    weight: 250
-isTocNested: true
-showAsideToc: true
```

Comment:
Are you sure you don't want the on-page TOC? I think you should keep it...

Reply:
Missed to have the TOC set . Will include it. Thanks.